tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -239,15 +239,17 @@ function get_tdx_ovmf_path_from_distro() {
 
 function install_artifacts() {
 	echo "copying kata artifacts onto host"
-	cp -au /opt/kata-artifacts/opt/kata/* ${dest_dir}/
-	chmod +x ${dest_dir}/bin/*
-	[ -d ${dest_dir}/runtime-rs/bin ] && \
-		chmod +x ${dest_dir}/runtime-rs/bin/*
+	local host_install_dir="/host/${dest_dir}"
```

Comment:
I think we are already mounting `/host`, so we can use this okay? Ignorant question, but do we know if that might cause any issues like mounting `/opt/kata` is for users?

Reply:
Yep, we can use this exactly as we're already mounting `/host`.

You mean, if we still mount `/opt/kata` **and** follow this approach? From my tests there's no issue, but we may end up with `/opt/kata` not being used at all.

Reply:
The main reason I removed `/opt/kata` is because if we have that, that's yet another place we'd need to adjust in order to have INSTALLATTION_PREFIX properly working.